repose-aggregator/components/services/http-client-service/http-client-service-impl/src/main/java/org/openrepose/core/services/httpclient/impl/HttpConnectionPoolProvider.java

```diff
@@ -58,8 +62,8 @@
     private HttpConnectionPoolProvider() {
     }
 
-    public static HttpClient genClient(String configRoot, PoolType poolConf) {
-
+    public static HttpClient genClient(String configRoot, PoolType poolConf, OpenTracingService openTracingService) {
+        // to enable some of the new features available in Apache HTTP Client 4.3, updating to builder pattern
```

Comment:
pretty sure you didn't do this?

Reply:
:D ... i did ... then cursed myself and reverted ... except for the tiny comment ... will remove :)